docs/reference/commandline/build.md

```diff
@@ -94,11 +91,9 @@ Build Syntax Suffix             | Commit Used           | Build Context Used
 
 If you pass an URL to a remote tarball, the URL itself is sent to the daemon:
 
-Instead of specifying a context, you can pass a single Dockerfile in the `URL`
-or pipe the file in via `STDIN`. To pipe a Dockerfile from `STDIN`:
-
 ```bash
 $ docker build http://server/context.tar.gz
+```
```

Comment:
Is this OK now? I know there was some confusion about a commit to this file in docker.github.io repo.